### PR TITLE
Improves usability and repository structure

### PR DESCRIPTION
**Pull request by BenjaminRodenberg**
this is a non-breaking change. Let's merge it into master, before we introduce breaking changes like #18.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,108 @@
...
+
+0. Clone this repository:
+    ```bash
+    $ git clone https://github.com/precice/elastictube1d.git
+    ```
+
+1. Navigate into the `python` directory and execute the `Allrun` script to run both solver participants directly:
+    ```bash
+    $ cd python/ && ./Allrun
+    ```
```
Comment (IshaanDesai) [inline README.md:32]: Just tried this out and the simulation hangs on my system with the terminal showing:
```
Starting Fluid participant...
Starting Structure participant...

Waiting for participants to finish...
(you may run 'tail -f Fluid.log' in another terminal to check the progress)
Fluid participant finished. Waiting for Structure participant...
```
The Fluid.log contains:
```
(0) 15:09:24 [xml::XMLAttribute]:202 in readValueSpecific: ^[[31mERROR: ^[[0mString to Double error
```
The Structure.log contains:
```
(0) 15:09:23 [impl::SolverInterfaceImpl]:112 in configure: ^[[0mThis is preCICE version 1.6.1
(0) 15:09:23 [impl::SolverInterfaceImpl]:113 in configure: ^[[0mRevision info: v1.6.1-124-ged8e871c
(0) 15:09:23 [impl::SolverInterfaceImpl]:114 in configure: ^[[0mConfiguring preCICE with configuration: "./precice-config.xml"
(0) 15:09:23 [impl::SolverInterfaceImpl]:157 in configure: ^[[0mRun in coupling mode
(0) 15:09:23 [impl::SolverInterfaceImpl]:226 in initialize: ^[[0mSetting up master communication to coupling partner/s
```

**Review by IshaanDesai — CHANGES_REQUESTED** [review]
Python tube `Allrun` script appears to hang during execution

File: README.md (hunk 1 above)
Comment (fsimonis) [reply README.md:32]: This is most likely due to the used locale.

Can you try executing `export LANG=en_US.UTF-8` prior to running the solvers?

Comment (IshaanDesai) [reply README.md:32]: I tried `export LANG=en_US.UTF-8` before running the solvers using the `Allrun` script but I see the same behaviour as mentioned above.

Comment (IshaanDesai) [reply README.md:32]: `Allrun` script worked on my system after I ran `export LC_NUMERIC=en_US.UTF-8`

**Review by IshaanDesai — APPROVED** [review]

**Pull request merged**